ArduPlane/tailsitter.cpp

```diff
@@ -634,6 +670,25 @@ void Tailsitter::speed_scaling(void)
             if (is_positive(sq_outflow)) {
                 spd_scaler = constrain_float(sq_hover_outflow / sq_outflow, gain_scaling_min.get(), throttle_scale_max.get());
             }
+
+            if (is_positive(disk_loading_min_outflow)) {
+                // calculate throttle required to give minimum outflow speed over control surfaces
+                if (is_positive(airspeed)) {
+                    disk_loading_min_throttle = (((sq(disk_loading_min_outflow) - sq(airspeed)) * (0.5 * rho)) / (disk_loading.get() * GRAVITY_MSS)) * hover_throttle;
+                } else {
+                    // estimate backwards airspeed
+                    float reverse_airspeed = 0.0;
+                    Vector3f vel;
+                    if (quadplane.ahrs.get_velocity_NED(vel)) {
+                        reverse_airspeed = quadplane.ahrs.earth_to_body(vel - quadplane.ahrs.wind_estimate()).x;
```

Comment:
I wonder if we might see some benefit using this same estimate for revere airspeed for control surface scaling rather than just throttle level. 

Its a tricky one, since were calculating a squared value, we cant assume a positive answer. We would end up with control surface inversion at some point. Of course that is physically the correct thing to do, however would end up with a threshold that moves all the servos from min one way and may the other. Oscillating about that point will be nasty. It is unlikely the PIDs will be tuned right for flying backwards in anycase. 

Hopefully with this min outflow speed we should never end up with negative airflow on the control surfaces in anycase.